features/docs/defining_steps/ambiguous_steps.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Ambiguous Steps
+ 
+  Scenario: Ambiguous steps
+
+    Given a file named "features/ambiguous.feature" with:
+    """
+    Feature:
+
+      Scenario: test 1
+      * a step
+      * an ambiguous step
+
+      Scenario: test 2
```

Comment:
@matt it's there because the current behaviour is that it just crashes the whole test run and doesn't run he second scenario. 

I suppose that if it reported the status correctly it wouldn't matter how many scenarios there are.